edbdeploy/data/terraform/azure/environments/vm/vm.tf

```diff
@@ -333,6 +480,43 @@ resource "azurerm_linux_virtual_machine" "pem_server" {
   tags = var.project_tags
 }
 
+resource "azurerm_linux_virtual_machine" "bdr_witness_server" {
+  count               = var.bdr_witness_server["count"]
+  name                = format("%s-%s%s", var.cluster_name, "bdr-witness", count.index + 1)
+  resource_group_name = var.resourcegroup_name
+  location            = var.azure_region
+
+  size           = var.pem_server["instance_type"]
+  admin_username = var.ssh_user
+
+  network_interface_ids = [element(azurerm_network_interface.bdr_witness_public_nic.*.id, count.index)]
+
+  admin_ssh_key {
+    username   = var.ssh_user
+    public_key = file(var.ssh_pub_key)
+  }
+
+  identity {
+    type = "SystemAssigned"
+  }
+
+  source_image_reference {
+    publisher = var.azure_publisher
+    offer     = var.azure_offer
+    sku       = var.azure_sku
+
+    version = "latest"
+  }
+
+  os_disk {
+    name                 = format("bdr-witness-%s-%s-%s", var.cluster_name, "EDB-VM-OS-Disk", count.index)
+    storage_account_type = var.pem_server["volume"]["storage_account_type"]
```

Comment:
Wrong variable name, should be `var.bdr_witness_server["volume"]["storage_account_type"]`

Reply:
Fixed